packages/storybook-addon/src/decorator/preact.ts

```diff
@@ -0,0 +1,16 @@
+import { addDecorator } from '@storybook/preact';
+import { Provider, createClient } from '@urql/preact';
+import { createElement } from 'preact';
+import { getStorybookExchange } from '../exchange';
+
+export const urqlDecorator: Parameters<typeof addDecorator>[0] = (
+  Story,
+  context
+) => {
+  const client = createClient({
+    url: 'storehhh',
```

Comment:
Let's keep this to `/graphql` maybe? 😆

Reply:
RIP the northern easter egg

```suggestion
    url: '/graphql',
```